e2e/browser/test-app/components/accessGrants/index.tsx

```diff
@@ -20,9 +20,12 @@
 //
 
 import { getDefaultSession } from "@inrupt/solid-client-authn-browser";
+
 import {
   approveAccessRequest,
   revokeAccessGrant,
+  redirectToAccessManagementUi,
+  getAccessGrant,
 } from "@inrupt/solid-client-access-grants";
```

Comment:
I'm surprised by these warnings. Are they reproducible locally, or CI-specific?

Reply:
the `unable to resolve path to module`s  do not show up locally. 

However I can remove a console.log or two.